premailer/__init__.py

```diff
@@ -1,4 +1,4 @@
 from __future__ import absolute_import, unicode_literals
 from .premailer import Premailer, transform  # noqa
 
-__version__ = "3.6.0"
+__version__ = "3.6.1"
```

Comment:
```suggestion
__version__ = "3.6.0"
```
I'll update that line when I make a release next.